src/components/detailedProduct/detailedProductView.ts

```diff
@@ -1,52 +1,37 @@
-import slider1 from '../../assets/slider-1.jpg';
-import slider2 from '../../assets/slider-2.jpg';
-import slider3 from '../../assets/slider-3.jpg';
-import slider4 from '../../assets/slider-4.jpg';
-import slider5 from '../../assets/slider-5.jpg';
-import slider6 from '../../assets/slider-6.jpg';
+import DetailedProductController from './detailedProductController';
 
 export default class DetailedProduct {
-  name: string = 'Product';
+  controller: DetailedProductController;
 
-  price: string = 'Price';
+  id: string;
 
-  discountPrice: string = 'Discount Price';
-
-  descriptionOfProduct: string = 'Description';
+  constructor(id: string) {
+    this.controller = new DetailedProductController();
+    this.id = id;
+  }
 
   renderMain() {
     const detailedProductWrapper = HTMLCreator.createElement('div', { class: 'detailed__product-wrapper' });
     const main = HTMLCreator.createElement('main', { class: 'detailed__product' }, [
       HTMLCreator.createElement('div', { class: 'detailed__product-container' }, [detailedProductWrapper]),
     ]);
 
-    const createSwiperSlide = (src: 'string') =>
-      HTMLCreator.createElement('picture', { class: 'swiper-slide', lazy: 'true' }, [
-        HTMLCreator.createElement('img', { src, alt: 'img', loading: 'lazy', 'data-fancybox': 'gallery' }),
-      ]);
-
-    const swiperWrapper = HTMLCreator.createElement('div', { class: 'swiper-wrapper' }, [
-      createSwiperSlide(slider1),
-      createSwiperSlide(slider2),
-      createSwiperSlide(slider3),
-      createSwiperSlide(slider4),
-      createSwiperSlide(slider5),
-      createSwiperSlide(slider6),
-    ]);
-
     const swiper = HTMLCreator.createElement('div', { class: 'swiper' }, [
-      swiperWrapper,
+      HTMLCreator.createElement('div', { class: 'swiper-wrapper' }),
       HTMLCreator.createElement('div', { class: 'swiper-pagination' }),
       HTMLCreator.createElement('div', { class: 'swiper-button-prev' }),
       HTMLCreator.createElement('div', { class: 'swiper-button-next' }),
     ]);
 
     const productContent = HTMLCreator.createElement('div', { class: 'detailed__product-content' }, [
-      HTMLCreator.createElement('h1', { class: 'detailed__product-title' }, [this.name]),
-      HTMLCreator.createElement('p', { class: 'detailed__product-description' }, [this.descriptionOfProduct]),
-      HTMLCreator.createElement('div', { class: 'detailed__product-price' }, [this.price]),
-      HTMLCreator.createElement('div', { class: 'detailed__product-discount' }, [this.discountPrice]),
+      HTMLCreator.createElement('h1', { class: 'detailed__product-title' }),
+      HTMLCreator.createElement('p', { class: 'detailed__product-description' }),
+      HTMLCreator.createElement('div', { class: 'detailed__product-inner' }, [
+        HTMLCreator.createElement('div', { class: 'detailed__product-price' }),
+        HTMLCreator.createElement('div', { class: 'detailed__product-discount' }),
+      ]),
+
```

Comment:
The productContent is built clearly and concisely. There are redundant elements being created but not utilized. Ensure all created elements are necessary for the rendering logic.